src/clusterfuzz/_internal/issue_management/issue_tracker_utils.py

```diff
@@ -43,7 +44,16 @@ def _get_issue_tracker_project_name(testcase=None):
   return data_handler.get_issue_tracker_name(job_type)
 
 
-@memoize.wrap(memoize.FifoInMemory(256))
+def request_or_task_cache(func):
+  """Cache that lasts for a bot task's lifetime,
```

Comment:
A bit of a weird format.

Reply:
@jonathanmetzman, just out of curiosity: What would be a more preferred format?
Thanks : )

Reply:
yeah, could you please add more context of this?

Reply:
I think Jonathan's just referring to the line lengths here? 

i.e. we can be fitting the docstring more into the first line and still fit it under 80 columns.

Reply:
The new line is formatted by butler.py format. Without the new line, pylint still fails even adding disable line-too-long.

Reply:
But I can split the string into two lines if preferred.

Reply:
Can we not do something like this:

```python
def request_or_task_cache(func):
  """Cache that lasts for a bot task's lifetime, or an App Engine request
  lifetime."""
  ...
```

?

Reply:
oh yeah sure. Sorry,  I was talking about another review comment.